### PR TITLE
Fix #110 - add HedgeDoc info in extra

### LINKED ISSUE #110: Add Extra about CodiMD
@annefou and I are going to use CodiMD for note-taking etc during the trial run next week, and (inspired by Code Refinery's [_HackMD Mechanics_](https://coderefinery.github.io/manuals/hackmd-mechanics/#basic-controls)) guide, we think it would be good to add an Extra about using CodiMD. This will probably help other instructors preparing to teach this lesson in their own workshops, and makes more sense as an Extra than a section in the Instructors' Notes because it will probably be fairly long...

### PR DESCRIPTION
**Pull request by annefou**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ title: "Instructor Notes"
 
 ## Introduction
 
+- We recommend the usage of [HedgeDoc](https://hedgedoc.org).
```
Comment (sstevens2) [inline _extras/guide.md]:
```suggestion
- We recommend the usage of [HedgeDoc](https://hedgedoc.org) (formerly called CodiMD).
```

File: _extras/guide.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ title: "Instructor Notes"
...
+  All instructors can use it for note-taking when preparing or teaching this lesson and we also encourage
+  its usage during the workshop for asking questions and collaborate during  exercises, especially when using breakout rooms.
```
Comment (sstevens2) [inline _extras/guide.md]:
```suggestion
  its usage during the workshop for asking questions and collaborate during exercises, especially when using breakout rooms.
```
extra space

File: _extras/guide.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ title: "Instructor Notes"
...
+  The main advantage is that it uses Markdown so it can easily convince learners on the importance of learning Markdown.
+  Make use of [HackMD Mechanics](https://coderefinery.github.io/manuals/hackmd-mechanics/#basic-controls) from [CodeRefinery](https://coderefinery.org/): it is important everyone understands how to use HedgeDoc from the very beginning of the workshop.
```
Comment (sstevens2) [inline _extras/guide.md]:
```suggestion
  Make use of [HackMD Mechanics](https://coderefinery.github.io/manuals/hackmd-mechanics/#basic-controls) from [CodeRefinery](https://coderefinery.org/). Take time to orient your learners to using the document early on to be ensure everyone understands how to use HedgeDoc.
```
Small wording suggestion.

**Review by sstevens2 — COMMENTED** [review]
I think what you have looks good @annefou.  I made a couple suggestions.  I was thinking maybe we should also add a link to using the Carpentries codimd instance somewhere?

**Comment by annefou** [comment]
> I think what you have looks good @annefou. I made a couple suggestions. I was thinking maybe we should also add a link to using the Carpentries codimd instance somewhere?

Should we suggest instructors to create new documents using https://codimd.carpentries.org/?

**Comment by sstevens2** [comment]
Yes, that is what I was thinking.  It seems like an accepted usage of the Carpentries instance based on some recent chat in the general slack channel.

**Comment by annefou** [comment]
Thank you @sstevens2 for your review!!!

**Comment by tobyhodges** [comment]
Thanks Anne!

**Comment by tobyhodges** [comment]
Closes #110

**Pull request merged**